packages/datagrid/src/components/DefaultHeaderRenderer/QualityBar.component.js

```diff
@@ -26,51 +26,118 @@ export function formatNumber(value) {
 
 function QualityBar(props) {
 	const { t = getDefaultT() } = useTranslation(I18N_DOMAIN_DATAGRID);
+	function getLibelleValue(numberOfItem) {
```

Comment:
as Romain said, we should do the changes in the new qualitybar
https://github.com/Talend/ui/tree/master/packages/components/src/QualityBar